packages/components/src/components/ScalarFloating/ScalarFloating.vue

```diff
@@ -76,7 +76,7 @@ const { floatingStyles, middlewareData } = useFloating(targetRef, floatingRef, {
     v-if="isOpen"
     :disabled="!teleport"
     to="body">
-    <div class="scalar-app">
+    <div class="scalar-app scalar-client">
```

Comment:
This probably shouldn't be hard coded into the components library 🤔 Could we add a prop or something?

Reply:
so `scalar-client` would be pass from the client app to the components using scalar floating under the hood then?

Reply:
Yeah exactly!